service/task/client.go

```diff
@@ -55,7 +55,20 @@ func (c *Client) GetByShortId(taskShortId string) (onfleet.Task, error) {
 	return task, err
 }
 
-func (c *Client) ListTasks(teamId string) {
+// Reference https://docs.onfleet.com/reference/list-tasks
+func (c *Client) ListPaginated(params onfleet.TaskListQueryParams) (onfleet.PaginatedTasks, error) {
```

Comment:
I'd suggest naming this simply List as user can understand from return type that LastId is/can be present for cases where multiple pages are present.